samples/subsys/nvs/boards/esp32c3_luatos_core.conf

```diff
@@ -0,0 +1 @@
+CONFIG_HEAP_MEM_POOL_SIZE=256
```

Comment:
I think you don't need to include this conf file as HEAP is set to default 4096. We could also in another PR remove other ESP32 board's conf from this sample. Same for the `_usb` conf file.

Reply:
I copy this file from esp32c3_devkitm.conf.

If you want, i can remove this file